satellite/accounting/projectusage.go

```diff
@@ -118,25 +117,25 @@ func (usage *ProjectUsage) ExceedsStorageUsage(ctx context.Context, projectID uu
 	}
 
 	maxUsage := limit.Int64() * int64(ExpansionFactor)
-	if inlineTotal+remoteTotal >= maxUsage {
+	if totalUsed >= maxUsage {
```

Comment:
I believe this function is used when creating a new segment and if `totalUsed == maxUsage`,  it will return an error and the segment will not be committed because the limit has been reached